doc/api/permissions.md

```diff
@@ -10,6 +10,12 @@ be accessed by other modules.
   This can be used to control what modules can be accessed by third-party
   dependencies, for example.
 
+* [Process-based permissions](#process-based-permissions) control the Node.js
+  process's access to resources (for now, only file system).
+  The resource can be entirely allowed or denied, or actions related to it can
+  be controlled. For example, file system reads can be allowed while denying
```

Comment:
Nit: Our docs tend to be a bit inconsistent on this... is it `file system` or `filesystem`?

Reply:
Well, @Trott might have the answer, but according to our docs, `file system` is more used.

```console
node on  feat/permission-system [$!?] via ⬢ v16.19.0
➜ grep -roh "filesystem" doc/ | wc -l
      25

node on  feat/permission-system [$!?] via ⬢ v16.19.0
➜ grep -roh "file system" doc/ | wc -l
      74
```

Reply:
We use the Microsoft Style Guide. It doesn't specify which one to use in its A-Z word list, but it does use "file system" internally, so I'd say let's go with that.

Example from https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/directory:

> Use _directory_ only in content for developers when you need to refer to the structure of the file system or to match the API.

Reply:
Updated. Thanks!